monai/transforms/utility/array.py

```diff
@@ -372,3 +373,39 @@ def __call__(self, img, func: Optional[Callable] = None):
             return self.func(img)
         else:
             raise RuntimeError("neither func or self.func is callable.")
+
+
+class LabelToMask(Transform):
+    """
+    Convert label to mask for other tasks.
```

Comment:
could you update the docstring, it's unclear when and how to use this transform. also perhaps `compose_channels` could be `merge_channels`

Reply:
updated in the latest commit.
Thanks.